### PR TITLE
add code climate configuration

### PR DESCRIPTION
**Pull request by vaidik**

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    enabled: true
+    config:
+      languages:
+        javascript:
+ratings:
+  paths:
+    - src/**
+    - "**.js"
+exclude_paths:
+  - "test/"
```
Comment (nimishsinghal) [inline .codeclimate.yml]: why are tests excluded?

**Comment by nimishsinghal** [comment]
Approved

File: .codeclimate.yml (hunk 1 above)
Comment (vaidik) [inline .codeclimate.yml]: Lesser worried at this time for tests climate. Also its possible that tests have some duplication in the beginning which is fine.

**Comment by amanmehta25** [comment]
Approved

**Pull request merged**